Uppsala/UppEt58.xml

```diff
@@ -0,0 +1,245 @@
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 58</idno>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        
+                        <msItem xml:id="ms_i1">
+                            <locus target="#1r"/>
+                            <title type="incomplete" ref="LIT2842RepCh125"/>
+                            <incipit xml:lang="gez">
+                                ሰላም ፡ ለገቦከ ፡ ኲናተ ፡ ለንጊኖስ ፡ ዘወግዖ ። ወ<surplus>አ</surplus>አዘቅተ ፡ ማይ ፡ 
+                                ከርሥከ ፡ ለደመ ፡ ሥርአት<surplus>ት</surplus>፡ እንተ ፡ አንቅዖ ፡ ሐዋርያ ፡ አብ ፡ 
+                                ክርስቶስ ፡ ዘፈጸምከ ፡ ግብረ ፡ ተሰብኦ ፡ ዕቀበኒ ፡ በመስቀልከ ፡ 
+                                ለመልአከ ፡ዓለም ፡ ዘሞዖ ፡ ከመ ፡ኖላዊ ፡ የዓቅብ፡ 
+                                እምተኵላ ፡ በግዖ ፡ ኢጢሚሶር ፡ ኢጢሚሶር ፡ ኢጢሚሶር ፡ 
+                                ኢጤሜሶር ፡ ኢጢሚሶር ፡ ኤጤሜሶር ፡ ኤጤሜሶር ፡ 
+                                ግድምናሶር ፡ ግድምናሶር ፡ ግድምናሶር ፡ ግድምናሶር ፡ 
+                                ግድምናሶር ፡ ግድምናሶር ፡ ልድሞናሶር ፡ ፈልዳማአስ ፡ 
+                                ፈልዳማአስ ፡ ፈልዳማአስ ፡ ፈልዳማአስ ፡ ፈልዳማአስ ፡ 
+                                ፈልዳማአስ ፡ ፈልዳማአስ ፡ ግቶሞር ፡ ግቶሞር ፡ግቶሞር ፡
+                                ግቶሞር ፡ ግቶሞር፡ ግቶሞርግቶሞር። <add place="above">በኃይለ፡</add> እሉ ፡ አስማቲከ፡ 
+                                ዕቀበ<hi rend="rubric">ኒ፡</hi> <add place="above">ወአድኅነኒ፡</add>
+                                <hi rend="rubric">
+                                    እምተ<unclear>ያ</unclear>ዥ፡ ወእምጠምዛዥ፡ እምቡዳ፡ ወእም<unclear>ሶ</unclear>ተሐምም፡ 
+                                    <gap reason="lost" unit="chars" quantity="1"/>
+                                    ገብርከ፡ እገሌ፡</hi><add place="above"><gap reason="lost" unit="chars" quantity="6"/>ሊተ፡ 
+                                        ለ<surplus>፡</surplus> እገሌ፡ </add>
+                            </incipit>
+                            <note>As identified by <bibl><ptr target="bm:alvin"/></bibl>, the work consists of the fifteenth stanza 
+                                of <ref type="work" corresp="LIT2842RepCh125"/> followed by a series 
+                                of <foreign xml:lang="gez">ʾasmāt</foreign>.</note>
```

Comment:
we have a keyword for `Asmat`, always nice to use it, to be able then from the keyword to find all manuscripts containing asmat